values-schema.yaml

```diff
@@ -1993,10 +1993,11 @@ properties:
       ingress-nginx-platform:
         $ref: '#/definitions/ingressNginx'
       istio:
-        additionalProperties: false
         properties:
           _rawValues:
             $ref: '#/definitions/rawValues'
+          defaultRevision:
```

Comment:
Do we really want users to define the default revision ?

Reply:
That was not the intention. But I needed a place to record the state of the transition from one to another. Also this allowed for going back and forth between the revisions, so technically, we could use this for upgrading to other revisions later.

The better source of truth of the migration state would clearly be the cluster, but the templates need this value. We could use Helmfile's `exec` to fetch it, if that seems like a better approach.

Reply:
Testing one more approach now, where I set the `defaultRevision` only during the time of the upgrade. Then it disappears from the values store again. That could avoid unintended changes.